suites/nautilus/rgw/tier-2_rgw_ganesha.yaml

```diff
@@ -0,0 +1,269 @@
+# Verify and run nfs-ganesha sanity tests
+---
+tests:
+  - test:
+      name: install ceph pre-requisities
+      module: install_prereq.py
+      abort-on-fail: true
+  - test:
+      name: ceph ansible
+      module: test_ansible.py
+      config:
+        ansi_config:
+          ceph_test: true
+          ceph_origin: distro
+          ceph_repository: rhcs
+          osd_scenario: lvm
+          osd_auto_discovery: false
+          journal_size: 1024
+          ceph_stable: true
+          ceph_stable_rh_storage: true
+          fetch_directory: ~/fetch
+          copy_admin_key: true
+          dashboard_enabled: false
+          ceph_conf_overrides:
+            global:
+              osd_pool_default_pg_num: 64
+              osd_default_pool_size: 2
+              osd_pool_default_pgp_num: 64
+              mon_max_pg_per_osd: 1024
```

Comment:
I do not think this is required for nfs-ganesha, you may remove these overrides.

Reply:
updated the changes.